app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def random_greeting
+    random_message = Message.order(Arel.sql('RANDOM()')).first
+    render json: { greeting: random_message.greeting }
+  end
```

Comment:
- [ ] Great job so far but I think there is an issue here in the random greeting controller since when I run it it shows me an error message so I highly recommend checking it and trying to fix it to let your app works as expected 🚀💯

- [ ] I tried to add some seeds as well to your app but also it doesn't work so kindly add some seeds to let the reviewer check on it 💯🚀
![image](https://github.com/Muhammad0602/hello-rails-back-end/assets/58553711/6febf8f2-445c-43ed-9535-c2f485258a6d)


## The error message 🚀

![image](https://github.com/Muhammad0602/hello-rails-back-end/assets/58553711/e6e9089c-1a1e-4912-ba12-291863c9dd26)

Reply:
Hello @Reem-lab, I don't know what you did, but on my side, everything is rendered as expected.

Reply:
I see that u r getting the error message, because you did not created any message and therefore it is nil

Reply:
I will try to add some seeds though. But the app should work properly.

Reply:
![image](https://github.com/Muhammad0602/hello-rails-back-end/assets/113791476/77797e95-c290-453f-aa6d-c4129e17abca)

Reply:
Now, I see your problem. The model for createing greetings is `Message`, not `random_greeting`. Random greeting is a method that I used in my controller.

Reply:
The right way to create a new greeting, would be 
` Message.create(greeting: "Hello")`